layouts/shortcodes/youtube-embed.html

```diff
@@ -1,4 +1,5 @@
-<div id="youtube-player-{{ .Get 0 }}" data-video-id="{{ .Get 0}}" class="youtube-video aspect-video w-full"></div>
+{{ $id := .Get 0 | default .Page.Params.yt }}
```

Comment:
I made this change so that I could programmatically render the youtube video on the overview page. Now that I am not doing that anymore, I will probably revert this change.